bedrock/firefox/views.py

```diff
@@ -606,18 +606,8 @@ def get_template_names(self):
             else:
                 template = 'firefox/whatsnew/index.html'
         elif channel == 'beta':
-            if version.startswith('74.'):
-                if locale in ['en-US', 'en-CA', 'en-GB']:
-                    template = 'firefox/whatsnew/whatsnew-fx70-en.html'
-                elif locale == 'de':
-                    template = 'firefox/whatsnew/whatsnew-fx70-de.html'
-                elif locale == 'fr':
-                    template = 'firefox/whatsnew/whatsnew-fx70-fr.html'
-                else:
-                    template = 'firefox/whatsnew/index.html'
-            else:
-                template = 'firefox/whatsnew/index.html'
-        elif locale == 'id' and show_firefox_lite_whatsnew(version):
+            template = 'firefox/whatsnew/index.html'
+        elif locale == 'id':
```

Comment:
Looks like you may have lost something in the rebase here?

```suggestion
        elif locale == 'id' and show_firefox_lite_whatsnew(version):
```